sdk/storage/Azure.Storage.Blobs/src/PartitionedUploader.cs

```diff
@@ -52,6 +68,7 @@ internal class PartitionedUploader
                     Constants.Blob.Block.MaxStageBytes,
                     transferOptions.MaximumTransferLength.Value);
             }
+            _operationName = operationName;
```

Comment:
Also need to thread this to the `_client.ClientDiagnostics.CreateScope(` calls later in this file

Reply:
Changed to ```_client.ClientDiagnostics.CreateScope(_operationName ?? $"{nameof(Azure)}.{nameof(Storage)}.{nameof(Blobs)}.{nameof(BlobClient)}.{nameof(BlobClient.Upload)}");```.